### ISSUE TITLE
Create "v1" tag for automatically following this action's updates without modifying workflows

### ISSUE THREAD

**Issue by rhysd** [actions/add-to-project#563]
I'd like to suggest to create a `v1` tag for making users to automatically follow `v1.x.y` updates without modifying their workflows like other actions in `actions` organization (e.g. [`actions/checkout@v4`](https://github.com/actions/checkout/releases/tag/v4)).

I know that Dependabot can automatically maintain the versions of actions used in workflow. But they require PR reviews and some people (including me) would don't want to spare time for them.

**Comment by GuySartorelli** [comment]
I came here with the intention to create this exact issue.

**Comment by MasterOdin** [comment]
The `actions/setup-node` repo uses https://github.com/actions/publish-action which looks like it makes it pretty simple to set this up: https://github.com/actions/setup-node/blob/26961cf329f22f6837d5f54c3efd76b480300ace/.github/workflows/release-new-action-version.yml

**Comment by boneskull** [comment]
FWIW, this is contrary to [GitHub's own recommendations](https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-third-party-actions). Even if you decide you trust the actions team, pinning to a SHA would keep you safe from [attacks like this](https://www.wiz.io/blog/github-action-tj-actions-changed-files-supply-chain-attack-cve-2025-30066).

**Comment by jgravois** [comment]
at this point a v2/v2.0.0 tag would be helpful.

its been 137 commits and almost 2 years since a release and v1.0.2 now results in a deprecation warning.

> [example](https://github.com/cal-itp/benefits/actions/runs/22970358802/job/66685107574#step:3:2)
Node.js 20 actions are deprecated. The following actions are running on Node.js 20 and may not work as expected: actions/add-to-project@v1.0.2. Actions will be forced to run with Node.js 24 by default starting June 2nd, 2026. Please check if updated versions of these actions are available that support Node.js 24. To opt into Node.js 24 now, set the FORCE_JAVASCRIPT_ACTIONS_TO_NODE24=true environment variable on the runner or in your workflow file. Once Node.js 24 becomes the default, you can temporarily opt out by setting ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION=true. For more information see: https://github.blog/changelog/2025-09-19-deprecation-of-node-20-on-github-actions-runners/

**Comment by jklare** [comment]
Hi @nasquasha :) I noticed that you are the one who recently merged a couple of RPs and was wondering if you are also involved in planning any upcoming release? If not, do you know who is responsible for that?

**Comment by nasquasha** [comment]
@jklare yes! We are aiming to get a release out soon. [Update to Node 24](https://github.com/actions/add-to-project/issues/786) was merged last week, v2 to follow. Thanks for your patience everyone, I know it's been ✨ a while ✨

**Comment by jklare** [comment]
@nasquasha thank you for the swift reply and pushing the release! Looking forward to it 🥳

**Comment by nasquasha** [comment]
done!
https://github.com/actions/add-to-project/releases/tag/v2.0.0
https://github.com/actions/add-to-project/releases/tag/v2

**Comment by jgravois** [comment]
thanks so much @nasquasha!

if you're able to isolate the breaking changes (only the node runtime update i think?) in the release notes for 2.0.0 it would help folks evaluate the potential impact in their own codebases at a glance. 🙏 

example: https://github.com/actions/setup-node/releases/tag/v6.0.0
